WebJobs.Script.sln

```diff
@@ -430,6 +430,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTriggerShared", "HttpTr
 		sample\HttpTriggerShared\function.json = sample\HttpTriggerShared\function.json
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ServiceBusListenerException", "ServiceBusListenerException", "{1AC76291-23CF-4698-8AC7-F8C27FCC9460}"
+	ProjectSection(SolutionItems) = preProject
+		sample\ServiceBusListenerException\function.json = sample\ServiceBusListenerException\function.json
```

Comment:
yeah good point. Had the misconception that the E2E tests were directly calling the `Host.Call` method and circumventing start/listener logic, but it doesn't seem like thats the case after looking closer at the E2E fixture